esp-radio/src/ble/os_adapter_esp32s3.rs

```diff
@@ -150,8 +152,19 @@ pub(super) static G_OSI_FUNCS: osi_funcs_s = osi_funcs_s {
     btdm_rom_table_ready: Some(btdm_rom_table_ready_wrapper),
     coex_bt_wakeup_request: Some(coex_bt_wakeup_request),
     coex_bt_wakeup_request_end: Some(coex_bt_wakeup_request_end),
+    get_time_us: Some(get_time_us_wrapper),
+    assert: Some(assert_wrapper),
 };
 
+extern "C" fn get_time_us_wrapper() -> u64 {
```

Comment:
Why do we have copies of these :(

Reply:
We currently have a lot of copies in the `ble/os_adapter_XXX.rs` - we should introduce a common adapter like we have for wifi - it's a bit unfortunate that BLE is subtle different for different chips

Reply:
but C3 + S3 are so similar that the ESP-IDF code is shared for them - we could probably do the same